src/main/scala/org/broadinstitute/dsde/firecloud/webservice/StaticNotebooksApiService.scala

```diff
@@ -13,12 +13,16 @@ trait StaticNotebooksApiService extends HttpService
   private implicit val executionContext = actorRefFactory.dispatcher
 
   val calhounStaticNotebooksRoot: String = FireCloudConfig.StaticNotebooks.baseUrl
+  val calhounStaticNotebooksURL: String = s"$calhounStaticNotebooksRoot/api/convert"
 
   val staticNotebooksRoutes: Route = {
     path("staticNotebooks" / "convert") {
       requireUserInfo() { _ =>
         post {
-          passthrough(s"$calhounStaticNotebooksRoot/api/convert", HttpMethods.POST)
```

Comment:
`passthrough()` automatically wraps a `respondWithMediaType(`application/json`)` directive, which we do not want.